### PR TITLE
Testing instructions

### PR DESCRIPTION
**Pull request by mirellel**
- added testing instructions in documentation/testing.md
- updated README to have a link to testing instructions and deleted the testing instructions from README

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -9,25 +9,7 @@
...
-
-## Get unit test coverage
-- Go to app/
-
-- Run ```poetry run invoke coverage```
-
-HTML-report is produced automatically.
-
-The report ```index.html``` is located in the htmlcov directory.
+- [Testing instuctions](https://github.com/ILMOWEB/ilmo/blob/main/documentation/testing.md)
```
Comment (sohvip) [inline README.md:12]: *instructions (typo)

File: documentation/testing.md [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+
+Example:
+
+![image](https://github.com/ILMOWEB/ilmo/assets/101889891/49fd5be9-a1ee-4eb7-a9b8-b00ddb975e2f)
+
+## Coverage
+To get test coverage run ```invoke coverage```
+Then access index.html file in the htmlcov folder /ilmo/app/htmlcov/
+
+Currert coverage:
```
Comment (sohvip) [inline documentation/testing.md:26]: maybe it should be current?

**Review by sohvip — CHANGES_REQUESTED** [review]
fix the typos, then it's PERFECT

**Review by Noraelisa — APPROVED** [review]
Yes perfect! :)

**Review by Noraelisa — APPROVED** [review]

**Review by Noraelisa — APPROVED** [review]

**Review by Noraelisa — APPROVED** [review]

**Pull request merged**